tests/Nox.ClientApi.Tests/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/Presentation.Api.OData.ODataServiceCollectionExtensions.g.cs

```diff
@@ -46,6 +47,12 @@ public static void AddNoxOdata(this IServiceCollection services)
         builder.EntityType<WorkplaceDto>();
         builder.EntityType<WorkplaceKeyDto>();
 
+        builder.EntitySet<StoreOwnerDto>("StoreOwners");
+
+        builder.EntityType<StoreOwnerDto>();
+        builder.EntityType<StoreOwnerKeyDto>();
+        builder.EntityType<StoreOwnerDto>().Ignore(e => e.DeletedAtUtc);
```

Comment:
Instead of ignoring the DeleteAtUtc property, why don't we just remove it from the Dto if we don't need it?

Reply:
I'm not 100% sure if this can be removed, will need to discuss it with @rochar afterwards